tests/bluetooth/audio/cap_commander/src/test_volume.c

```diff
@@ -0,0 +1,552 @@
+/* test_change_volume.c - unit test for volume settings */
+
+/*
+ * Copyright (c) 2023-2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
+
+#include <zephyr/bluetooth/audio/cap.h>
+#include <zephyr/bluetooth/audio/vcp.h>
+#include <zephyr/fff.h>
+
+#include "bluetooth.h"
+#include "cap_commander.h"
+#include "conn.h"
+#include "expects_util.h"
+#include "cap_mocks.h"
+
+#define FFF_GLOBALS
+
```

Comment:
Is it only one of the files that need to implement the `ZTEST_RULE`? 

The VCP and VOCS init and cleanup sounds like they belong here, rather than in the `main` file. 

The include of `vcp.h` in the `main.c` file should also be removed, right?

Reply:
I'll change the implementation so that it is similar to the ascs unit-test